packages/cloud-cognitive/src/components/Datagrid/styles/_useInlineEdit.scss

```diff
@@ -0,0 +1,166 @@
+/*
+* Licensed Materials - Property of IBM
+* 5724-Q36
+* (c) Copyright IBM Corp. 2021
+* US Government Users Restricted Rights - Use, duplication or disclosure
+* restricted by GSA ADP Schedule Contract with IBM Corp.
+*/
+
+@import '../../../global/styles/project-settings';
+@import './variables';
+
+$row-heights: (
+  xs: 1.5rem,
+  sm: 2rem,
+  md: 2.5rem,
```

Comment:
I don’t mind us keeping this in here, but based on our [data table settings](https://pages.github.ibm.com/cdai-design/pal/components/data-table/settings/usage/#overview), `md` is just Carbon 11, although this way we’re less likely to miss this.